app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.spec.tsx

```diff
@@ -39,7 +39,7 @@ const USPS_RESPONSE = [
 const DEFAULT_PROPS = {
   toPreviousStep() {},
   onChange() {},
-  registerField() {},
+  registerField: (_string) => undefined,
```

Comment:
If we're going to list the unused argument, I'd think we should name it the same as it's named in the original type signature (`field`). There's also technically an `options` parameter as well if we wanted to be exhaustive.

Personally, I don't think it matters much to list the arguments, which is why I left it out in my previous recommendations.

https://github.com/18F/identity-idp/blob/c3863647960ebc1151d317ffbd809fe4e40446d8/app/javascript/packages/form-steps/form-steps.tsx#L33-L34